Prox/Prox/Data/PlacesResult.swift

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+private enum DeferredResult {
+    case success
+    case failure
+}
+
+class PlacesResult {
```

Comment:
nit: I'd add a comment to explain why we do this. Maybe take the one from before?